src/components_new/DocumentEditor.tsx

```
@@ -0,0 +1,1030 @@
+      setIsMobile(window.innerWidth < 768);
+      if (window.innerWidth < 768) {
+        setShowSidebar(false);
+      }
+    };
+    
+    checkMobile();
+    window.addEventListener('resize', checkMobile);
+    return () => window.removeEventListener('resize', checkMobile);
+  }, []);
+  
+  const { setEditor, setUndoManager } = useEditorStore();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'setUndoManager'. Consider to delete it or rename it.

'setUndoManager' is defined here:


(lint/suspicious/noRedeclare)